exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/StringFunctions.java

```diff
@@ -1280,6 +1280,43 @@ public void eval() {
     } // end of eval
   }
 
+  @FunctionTemplate(name = "split", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
+  public static class Split implements DrillSimpleFunc {
+    @Param  VarCharHolder input;
+    @Param  VarCharHolder delimiter;
+
+    @Workspace char splitChar;
```

Comment:
How about reusing the Splitter?

Create a workspace splitter once in setup, and use in eval.